ncurses/plan.sh

```diff
@@ -1,6 +1,6 @@
 pkg_name=ncurses
 pkg_origin=core
-pkg_version=6.0
+pkg_version=5.0
```

Comment:
@bdangit you can open a PR with an ncurses 5 package if you need it! Place the plan in an ncurses5 directory and make sure pkg_name is set to ncurses5